aii-pxelinux/src/test/perl/pxelinux_ks_hooks.t

```diff
@@ -0,0 +1,53 @@
+use strict;
+use warnings;
+use Test::More;
+use Test::Quattor qw(pxelinux_ks_hooks);
+use NCM::Component::pxelinux;
+use CAF::FileWriter;
+use CAF::Object;
+use Data::Dumper;
+use constant RESCUE_HOOK_PATH => '/system/aii/hooks/rescue';
+use constant INSTALL_HOOK_PATH => '/system/aii/hooks/install';
+use constant REMOVE_HOOK_PATH => '/system/aii/hooks/remove';
+use constant CONFIGURE_HOOK_PATH => '/system/aii/hooks/configure';
+use constant BOOT_HOOK_PATH => '/system/aii/hooks/boot';
+use constant FIRMWARE_HOOK_PATH => '/system/aii/hooks/firmware';
+use constant LIVECD_HOOK_PATH => '/system/aii/hooks/livecd';
+
+=pod
+
+=head1 SYNOPSIS
+
+Tests for the configure hook
+
+=cut
+
+$CAF::Object::NoAction = 1;
+
+
+# mock filepath, it has this_app->option
+my $ksuserhooks;
+my $fp = "target/test/pxelinux";
+my $mockpxe = Test::MockModule->new('NCM::Component::pxelinux');
+$mockpxe->mock('filepath', $fp);
+
+my $ks = NCM::Component::pxelinux->new();
+my $cfg = get_config_for_profile('pxelinux_ks_hooks');
+
+NCM::Component::pxelinux::pxeprint($cfg);
+$ksuserhooks = NCM::Component::pxelinux::ksuserhooks($cfg, CONFIGURE_HOOK_PATH);
+is($ksuserhooks, undef, 'Configure hook');
```

Comment:
open an issue to write improved unittests for the AII hooks, not ot delay this PR
@jrha ok with this?